packages/account-v2/src/components/SignupWizard/index.tsx

```diff
@@ -31,7 +55,9 @@ const SignupWizard: React.FC = () => {
             style={CUSTOM_STYLES}
         >
             <div className='bg-background-primary-base h-[71.7rem] w-[104rem] rounded-800 flex overflow-hidden'>
-                <div className='min-w-[25.6rem] bg-[#f2f3f4] p-800'>Timeline</div>
+                <div className='min-w-[25.6rem] bg-[#f2f3f4] p-800'>
```

Comment:
@shahzaib-deriv `#f2f3f4` color in `tradershub` package, for example, is `secondary-background`, in my opinion, better to add this color as var to `tailwind.config` and use it.

Reply:
We're gonna move the whole flow to traderhub and I'll use the color from our config. Thanks for the suggestion